recipes/tabulate/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "tabulate" %}
+{% set version = " v1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/p-ranav/tabulate/archive/{{ version }}.tar.gz
+  sha256: 34eda3605e2ce0c7ca339fa2af6a0c77a5e49321b2f549ff5313f955273c8ac8
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    run_exports:
+      - {{ pin_subpackage(name, max_pin='x.x') }}
+  host:
+    - tabulate
+
+test:
+
```

Comment:
Don't really understood how xeus-cling is [doing](https://github.com/conda-forge/xeus-cling-feedstock/blob/d3324e5eedf64d3cad803c0e3db4b9a635205794/recipe/meta.yaml#L49-L55) it to the point that I can reproduce it hehe
Could you please shed some light here?

Reply:
in this case it's much more advisable to copy xtensor, rather than xeus-cling. 

xtensor is also a header only library.

in the xtensor tests, we're just checking that the headers are installed into the correct locatoin (as well as CMake file).

https://github.com/conda-forge/xtensor-feedstock/blob/3869da64ec39013d78375ce11f62b6864060592b/recipe/meta.yaml#L27-L36

Note: on unix systems `test -f somefile.txt` checks if a file exists, and `test -d mydir/` checks that a directory exists.

Reply:
Thanks, Wolf! :)